commit 11111f59e19871b6ce17696a29c64fcd8fec3dd8
Author: Peter Robinson <pbrobinson@gmail.com>
Date:   Sun May 5 21:07:12 2019 +0100

    arm: dts: overlays: rpi-sense: add upstream humidity compatible
    
    The upstream humidiity driver uses "st,hts221" for the compatible
    string so add that in as well so it will work with an unmodified
    upstream kernel driver. We leave the downstream as the priority.
    
    Signed-off-by: Peter Robinson <pbrobinson@gmail.com>

diff --git a/arch/arm/boot/dts/overlays/rpi-sense-overlay.dts b/arch/arm/boot/dts/overlays/rpi-sense-overlay.dts
index 27153240e1be..6eba2a10bfa5 100644
--- a/arch/arm/boot/dts/overlays/rpi-sense-overlay.dts
+++ b/arch/arm/boot/dts/overlays/rpi-sense-overlay.dts
@@ -38,7 +38,7 @@
 			};
 
 			hts221-humid@5f {
-				compatible = "st,hts221-humid";
+				compatible = "st,hts221-humid", "st,hts221";
 				reg = <0x5f>;
 				status = "okay";
 			};
diff -uN a/1.txt b/1.txt
--- a/dummy/rpi_1493_11111f59e19871b6ce17696a29c64fcd8fec3dd8.txt	1970-01-01 00:00:00.000000000 +0000
+++ b/dummy/rpi_1493_11111f59e19871b6ce17696a29c64fcd8fec3dd8.txt	2013-12-23 04:07:40.000000000 +0000
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
